flambe/sampler/base.py

```diff
@@ -177,7 +177,7 @@ def collate_fn(data: List[Tuple[torch.Tensor, ...]],
             sizes = [tensor.size() for tensor in tensors]
 
             if all(s == sizes[0] for s in sizes):
-                stacked_tensors = torch.stack(tensors).squeeze(1)
+                stacked_tensors = torch.stack(tensors)
```

Comment:
Can you explain what this bug fixes? Is it for single channel images? To be honest, I'm not so sure if this squeeze is necessary.. I vaguely recall doing it for a reason, but I didn't add any comment, so hard to know now. Can you make sure the examples in the example folder run ok?

Reply:
Oh it was for labels, which you seem to have addressed above